src/node/extensions.ts

```diff
@@ -0,0 +1,50 @@
+import fs from "fs";
```

Comment:
Should this be `import * as fs from 'fs'` for correctness?

Reply:
Nope. I've configured typescript for more readable imports like this. 😄